lib/class-wp-rest-menu-items-controller.php

```diff
@@ -438,7 +497,8 @@ protected function prepare_item_for_database( $request ) {
 		if ( '' !== $prepared_nav_item['menu-item-url'] ) {
 			$prepared_nav_item['menu-item-url'] = esc_url_raw( $prepared_nav_item['menu-item-url'] );
 			if ( '' === $prepared_nav_item['menu-item-url'] ) {
-				return new WP_Error( 'invalid_url', __( 'Invalid URL.' ) ); // Fail sanitization if URL is invalid.
+				// Fail sanitization if URL is invalid.
+				return new WP_Error( 'invalid_url', __( 'Invalid URL.' ) );
```

Comment:
Should we be sending a 400 status here?

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/382cb9f354e1cce22f73e92c37f3f51f774c73bd